vignettes/VectorControl.Rmd

```diff
@@ -117,15 +126,42 @@ sprayingparams <- set_spraying(
   sprayingparams,
   timesteps = spraying_events$timestep,
   coverages = rep(.8, 2),
-  ls_theta = matrix(c(2.025, 2.025), nrow=2, ncol=1),
-  ls_gamma = matrix(c(-0.009, -0.009), nrow=2, ncol=1),
-  ks_theta = matrix(c(-2.222, -2.222), nrow=2, ncol=1),
-  ks_gamma = matrix(c(0.008, 0.008), nrow=2, ncol=1),
-  ms_theta = matrix(c(-1.232, -1.232), nrow=2, ncol=1),
-  ms_gamma = matrix(c(-0.009, -0.009), nrow=2, ncol=1)
+  ls_theta = matrix(c(2.025, 2.025), nrow=2, ncol=3),
+  ls_gamma = matrix(c(-0.009, -0.009), nrow=2, ncol=3),
+  ks_theta = matrix(c(-2.222, -2.222), nrow=2, ncol=3),
+  ks_gamma = matrix(c(0.008, 0.008), nrow=2, ncol=3),
+  ms_theta = matrix(c(-1.232, -1.232), nrow=2, ncol=3),
+  ms_gamma = matrix(c(-0.009, -0.009), nrow=2, ncol=3)
 )
 
 output <- run_simulation(sim_length, sprayingparams)
 
 add_intervention_lines(plot_prevalence(output), spraying_events)
 ```
+
+
+## Larval habitat management
+
+We can do the same for LSM with the individual_mosquitoes parameter set to FALSE ...
+This functionality is not yet operatable for the individual mosquito version of the model.
+
+```{r}
+habitat_managementparams <- simparams
+
+lsm_events = data.frame(
+  timestep = 3 * year, ## must be a single time point
+  name=c("LSM on")
+)
+
+habitat_managementparams <- set_habitat_management(
+  habitat_managementparams,
+  timesteps = lsm_events$timestep,
+  larvi_min = matrix(0.2,nrow=1,ncol=3),
+  lsm_rate = matrix(4,nrow=1,ncol=3),
+  deprec_param = matrix(-1,nrow=1,ncol=3)
+)
+
```

Comment:
I've just been playing with this example a bit. 
When I set `larvi_min` to be 0, which I think is a 100% reduction in laval habitats, I expected to see a very quick crash of the whole population. Instead I see a slow decline over a period much greater than the mosquito lifspan (~ 1000 days). Changing the `lsm_rate` and `deprec_param` up or down an order of magnitude doesn't seem to impact this much.
I'm a bit puzzled - but perhaps am not just understanding...

Reply:
This was a mistake - the limits for the larvi_min (now lsm_new_eqm) parameter can be -1 to 1. Similarly, the function was initially written with exp(-4) as the lsm_rate and this is now corrected. Have added defaults and recommended limits for the parameters.